README.md

```diff
@@ -4,60 +4,150 @@
-We're working on a polished release which will include more documentation, and more maintainable instances. We expect this to land
-by early July.
+- `sourcecred load`: Regenerate the cache
+- `sourcecred graph`: Recompute graphs from cache
+- `sourcecred score`: Re-run Cred calculations
+- `sourcecred site`: Regenerate the website (potentially upgrading it)
+- `sourcecred go`: Runs `load`, `graph` and `score` in sequence.
 
-For now, if you want to use SourceCred, you might start by forking [MetaGame's Cred Instance].
+If you want to update the data for just one plugin (e.g. `sourcecred/github`), you can use the following
+command sequence:
 
-Note that our next release (v0.7.0) will totally revamp how SourceCred instances are setup, and replace the CLI. As such,
-expect that migrating from v0.6.0 to v0.7.0 will involve making changes to your configuration.
+- `sourcecred load sourcecred/github`
+- `sourcecred graph sourcecred/github`
+- `sourcecred score`
 
-[MetaGame's Cred Instance]: https://github.com/MetaFam/TheSource
+## Contributing Guidelines
+
+If you'd like to contribute to the codebase, we ask you to follow the following
+steps:
+
+### 1. Drop by [our Discord].
+
+Come to the #intros channel and introduce yourself. Let us know that you're
+interested in helping out. We're friendly and will be happy to help you get
+oriented.
+
+### 2. Read our [Contributing Guidelines].
+
+We pride ourself on tidy software engineering; part of how we do that is by
+splitting our changes up into many small, atomic commits, each of which are
+easy to review. If you'd like to work alongside us, we ask you to adopt our
+practices.
+
+### 3. Find an issue to work on.
+
+You can check out the issues marked [contributions welcome], or ask in the
+Discord's #programming channel if anyone has something you can contribute to.
+
+[our Discord]: https://sourcecred.io/discord
+[Contributing Guidelines]: ./contributing.md
+[contributions welcome]: https://github.com/sourcecred/sourcecred/issues?q=is%3Aopen+is%3Aissue+label%3Acontributions-welcome
 
 ## Development Setup
 
+### Dependencies
+
   - Install [Node] (tested on v12.x.x and v10.x.x).
   - Install [Yarn] (tested on v1.7.0).
-  - Create a [GitHub API token]. No special permissions are required.
   - For macOS users: Ensure that your environment provides GNU
     coreutils. [See this comment for details about what, how, and
     why.][macos-gnu]
 
 [Node]: https://nodejs.org/en/
 [Yarn]: https://yarnpkg.com/lang/en/
-[GitHub API token]: https://github.com/settings/tokens
 [macos-gnu]: https://github.com/sourcecred/sourcecred/issues/698#issuecomment-417202213
 
+If you want to work on the GitHub plugin, you should
+create a [GitHub API token]. No special permissions are required.
+
+[GitHub API token]: https://github.com/settings/tokens
+
+Then, set it in your environment:
+```Bash
+export SOURCECRED_GITHUB_TOKEN=1234....asdf
+```
```

Comment:
Add instructions for Discord API Token

Reply:
Added a link to the instructions here: https://github.com/sourcecred/example-instance#discord